doc/user_guide/eds.rst

```diff
@@ -543,50 +561,81 @@ The obtained composition is in atomic percent, by default. However, it can be tr
     Fe (Fe_Ka): Composition = 4.96 weight percent
     Pt (Pt_La): Composition = 95.04 weight percent
 
-or with :py:func:`~.misc.material.atomic_to_weight`. The reverse method is :py:func:`~.misc.material.weight_to_atomic`.
+or using :py:func:`~.misc.material.atomic_to_weight`:
 
 .. code-block:: python
 
     >>> # With atomic_percent from before
     >>> weight_percent = hs.material.atomic_to_weight(atomic_percent)
 
-The zeta-factor method needs both the 'beam_current' (in nA) and the acquisition or dwell time (referred to as 'real_time' in seconds) in order to provide an accurate quantification. Both of the these parameters can be assigned to the metadata of the spectrum using:
+The reverse method is :py:func:`~.misc.material.weight_to_atomic`.
+
+The zeta-factor method needs both the 'beam_current' (in nA) and the acquisition
+or dwell time (referred to as 'real_time' in seconds) in order to obtain an accurate
+quantification. Both of the these parameters can be assigned to the metadata using:
 
 ..code-block:: python
```

Comment:
Now that you're at it you could also fix the missing space here, it should be: `.. code-block:: python`